src/pages/AccessibilityStatement.svelte

```diff
@@ -0,0 +1,120 @@
+<script lang="ts">
+  import "../style/main.css";
+  // @ts-expect-error no declarations
+  import { initAll } from "govuk-frontend";
+  import { AlphaBanner, DefaultButton } from "govuk-svelte";
+  import { Header } from "lib/common";
+  import { onMount } from "svelte";
+
+  onMount(async () => {
+    // For govuk components. Must happen here.
+    initAll();
+  });
+</script>
+
+<div class="govuk-width-container govuk-prose">
+  <Header />
+  <AlphaBanner />
+
+  <h1 style="margin-top: 30px">
+    Plan Your Active Travel Schemes Accessibility Statement
+  </h1>
+
+  <p>
+    This accessibility statement applies to the Plan Your Active Travel Schemes
+    tool, which is for sketching and viewing active travel schemes at various
+    stages of the bidding and development process.
+  </p>
+  <p>
+    This website is run by Active Travel England. It requires use of and
+    interactions with maps.
+  </p>
+  <ul>
+    <li>
+      We include multiple options for basemap, allowing the user to decide what
+      works best for them
+    </li>
+    <li>
+      We intend to add user-configurable colour choices for map layers such as
+      boundaries, sketched schemes and trip generators
+    </li>
+    <li>
+      <a href="https://mcmw.abilitynet.org.uk/">AbilityNet</a>
+       has advice on making your device easier to use if you have a disability
+    </li>
+  </ul>
+
+  <h2>How accessible this website is</h2>
+
+  <p>We know some parts of the website are not fully accessible:</p>
+  <p>
```

Comment:
This paragraph seems out of place. Did someone write this wording for us and we're just copying it in?

Reply:
I am copying the accessibility statement that was approved by DfT Accessibility guy